galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.docker.manager/src/main/java/dev/galasa/docker/IDockerContainer.java

```diff
@@ -42,6 +40,13 @@
 	 * @return a {@link IDockerImage} for this container - never null
 	 */
 	public IDockerImage getDockerImage();
+	
+	/**
+	 *  Fetch the docker containers tag
+	 *  
+	 *  @return a @string of the docker tag for this container
+	 */
+	public String getContainerTag();
```

Comment:
If a user is passing a Tag when they request a container, do we really need a method to return the data they have input?